enhancements/builds/disable-jenkins-pipeline.md

```diff
@@ -0,0 +1,249 @@
+---
+title: disable-jenkins-pipeline
+authors:
+  - "@adambkaplan"
+reviewers:
+  - @gabemontero
+  - Akram
+approvers:
+  - "@bparees"
+  - Jessica Forrester
+  - "@sbose78"
+creation-date: 2021-04-21
+last-updated: 2021-04-23
+status: provisional
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Disable Jenkins Pipeline Strategy
+
+**Note - this enhancement is provisional and is intended to solicit feedback**
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+OpenShift 3.0 included a way to invoke a Jenkins pipeline from an OpenShift build.
+The status of the build on the OpenShift cluster would reflect the state of the pipeline in Jenkins.
+This was the first feature to support continuous integration/delivery processes on OpenShift.
+
+As OpenShift and Kubernetes have evolved, so has cloud-native CI/CD.
+Tekton and its downstream distribution - OpenShift Pipelines - provide a way of running CI/CD processes natively on Kubernetes.
+The Jenkins pipeline strategy was officially deprecated in OpenShift 4.1, but at that time there was no meaningful replacement.
+We now have that meaningful replacement with the general availability of OpenShift Pipelines.
+It is time to disable OpenShift's direct integration with Jenkins.
+
+## Motivation
+
+Distrbuting and supporting Jenkins directly has always been challenging for OpenShift.
+Jenkins is based in Java and is extended by a convoluted ecosystem of plugins, which are a constant source of bugs and CVEs.
+Historically, the sprawling nature of Jenkins and its dependencies has made it impossible for Red Hat to independently compile Jenkins and the libraries used by OpenShift's plugin extensions.
+We instead distribute Jenkins directly from community binaries, making it vulnerable to supply chain attacks from sophisticated attackers.
+Given the increasing prevalence of supply chain attacks globally (SolarWinds), relying on third parties for binary code is becoming less tolerable for product security. 
```

Comment:
Historical reference - Openshift has had this able to build from jenkins binary exception since Openshift 2

yes that is 2, not 3 nor 4